src/model/row.py

```diff
@@ -0,0 +1,54 @@
+import uuid
+import os
+import json
+import pathlib
+from model.file_mode import *
+
+
+class Row:
+    def __init__(self, table, data):
+        self.table = table
+        self.primary_key = self.__get_primary_key__(data.get(table.get_primary_key()))
+        data[table.get_primary_key()] = self.primary_key
+        self.data = data
+
+    def __get_primary_key__(self, primary_key):
+        if primary_key is not None:
+            return primary_key
+        path = self.table.get_path()
+        while primary_key is None and os.path.exists(path):
```

Comment:
Why do you have a while here ? You shouldn’t have a while. UUID guarantee to be unique.

Reply:
I was mis understanding (.node) and i fixed it